docs/getting-started/installation.md

```diff
@@ -66,13 +66,13 @@ compiled dependencies found in multiple channels. An alternative is to install e
 initially creating the environment using, with AmberTools:
 
 ```shell
-mamba create -n bespokefit-env -c conda-forge  python=3.10 openff-bespokefit "qcportal <0.50" psi4 dftd3-python ambertools
+mamba create -n bespokefit-env -c conda-forge python=3.10 openff-bespokefit "qcportal >=0.50" psi4 dftd4-python ambertools
 ```
 
 or with OpenEye Toolkits:
 
 ```shell
-mamba create -n bespokefit-env -c conda-forge -c openeye python=3.10 openff-bespokefit "qcportal <0.50" psi4 dftd3-python openeye-toolkits
+mamba create -n bespokefit-env -c conda-forge -c openeye python=3.10 openff-bespokefit "qcportal >=0.50" psi4 dftd4-python openeye-toolkits
```

Comment:
These changes I'm deciding to leave in place - the Python version is less important than the `dftd4-python` change, which I understand to be essential

Reply:
Do you have a link for why this change is essential? My current (vague) understanding is that dftd3 and dftd4 have fundamentally different capabilities.

Reply:
This is a dependency update in Psi4: https://github.com/openforcefield/openff-bespokefit/pull/308#issuecomment-1841921111

Reply:
Ah, thanks for linking that context.

Haha I'm not sure this resolves it and I'm seeing we're having the same discussion as we had there.

I don't think `dftd4` is version 4 of the `dftd` package. I think `d3` and `d4` are fundamentally different things. 
BespokeFit's default level of theory is B3LYP-D3BJ/DZVP. I think the "D3" part of that alphabet soup maps to the `dftd3` package. 

Basically I think Lori is saying that back in the day, `psi4::dftd3` was packaged differently from `psi4::dftd4`. The latter was packaged more cleanly. Now, on `conda-forge`, both are packaged in a clean way. That is, `conda-forge::dftd4-python` and its deps are largely unchanged from their `psi4` channel versions. But `conda-forge::dftd3-python` and its deps are heavily changed from their `psi4` channel versions. 

Indeed, if I make a bespokefit env with the `dftd4-python` package (and not the dftd3 one), and then try to run a basic fit using the default level of theory:

```
openff-bespoke executor run --smiles 'CC' --workflow 'default' --target-torsion '[C:1]-[C:2]
```

I get an error that complains about missing dftd3

``` /Users/jeffreywagner/conda/envs/bespokefit-env/share/psi4/basis/dgauss-dzvp-mix.gbs \n    atoms 2, 5        entry C           
 line   192 file /Users/jeffreywagner/conda/envs/bespokefit-env/share/psi4/basis/dgauss-dzvp-mix.gbs \n\nTraceback (most recent
 call last):\n  File \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/qcengine/programs/base.py\",
 line 79, in get_program\n    ret.found(raise_error=True)\n  File                                                              
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/qcengine/programs/dftd_ng.py\", line 205, in    
 found\n    return which_import(\n           ^^^^^^^^^^^^^\n  File                                                             
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/qcelemental/util/importing.py\", line 45, in    
 which_import\n    raise ModuleNotFoundError(\nModuleNotFoundError: Python module 'dftd3' not found in envvar PYTHONPATH.      
 Please install a dftd3 version with enabled Python API (e.g. conda install dftd3-python -c conda-forge)\n\nThe above exception
 was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File                                
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/schema_wrapper.py\", line 460, in   
 run_qcschema\n    ret_data = run_json_qcschema(input_model.dict(), clean, False, keep_wfn=keep_wfn)\n                         
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File                                                
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/schema_wrapper.py\", line 626, in   
 run_json_qcschema\n    val, wfn = methods_dict_[json_data[\"driver\"]](method, **kwargs)\n                                    
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File                                                                  
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/driver.py\", line 638, in gradient\n
 wfn = procedures['gradient'](lowername, molecule=molecule, **kwargs)\n                                                        
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File                                             
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/procrouting/proc.py\", line 96, in  
 select_scf_gradient\n    return func(name, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^\n  File                                 
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/procrouting/proc.py\", line 2688, in
 run_scf_gradient\n    ref_wfn = run_scf(name, **kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^\n  File                        
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/procrouting/proc.py\", line 2588, in
 run_scf\n    scf_wfn = scf_helper(name, post_scf=False, **kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n 
 File \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/procrouting/proc.py\", line    
 1851, in scf_helper\n    disp_energy = scf_wfn._disp_functor.compute_energy(scf_wfn.molecule(), scf_wfn)\n                    
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File                                                     
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/psi4/driver/procrouting/empirical_dispersion.py\
 ", line 264, in compute_energy\n    jobrec = qcng.compute(\n             ^^^^^^^^^^^^^\n  File                                
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/qcengine/compute.py\", line 79, in compute\n    
 executor = get_program(program)\n               ^^^^^^^^^^^^^^^^^^^^\n  File                                                  
 \"/Users/jeffreywagner/conda/envs/bespokefit-env/lib/python3.11/site-packages/qcengine/programs/base.py\", line 81, in        
 get_program\n    raise ResourceError(f\"Program {name} is registered with QCEngine, but cannot be found.\") from              
 err\nqcengine.exceptions.ResourceError: Program s-dftd3 is registered with QCEngine, but cannot be found.\n\n\n
```

however directly installing `dftd3-python` resolves this error.

Reply:
I'm just going to remove all related changes as this is unrelated to the objective, they can be fixed elsewhere